### PR TITLE
msglist [nfc]: Remove some color computation in _UnreadMarker

### PR DESCRIPTION
**Pull request by chrisbobbe**
This seems like a hot enough codepath that it might benefit from these small optimizations.

### CODE REVIEW

File: lib/widgets/message_list.dart [hunk 1]
```diff
@@ -474,11 +474,11 @@ class _UnreadMarker extends StatelessWidget {
   final bool isRead;
   final Widget child;
 
-  // The color hsl(227deg 78% 59%) comes from the Figma mockup at:
+  // The color comes from the Figma mockup at:
   //   https://www.figma.com/file/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=132-9684
   // See discussion about design at:
   //   https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/flutter.3A.20unread.20marker/near/1658008
-  static final color = const HSLColor.fromAHSL(1, 227, 0.78, 0.59).toColor();
+  static const color = Color(0xff466ae8);
```
Comment (gnprice) [inline lib/widgets/message_list.dart:481]: A nice thing about statics is that their initializers only run once. So thankfully this isn't actually hot, and we can leave the source code in the HSL form which is somewhat more transparent to understand.

Comment (gnprice) [reply lib/widgets/message_list.dart:481]: OTOH if this did appear inside the build phase, it'd be good to pull it out as a static.

File: lib/widgets/message_list.dart [hunk 2]
```diff
@@ -496,14 +496,14 @@ class _UnreadMarker extends StatelessWidget {
...
-            child: DecoratedBox(
+            child: const DecoratedBox(
               decoration: BoxDecoration(
                 color: color,
                 // TODO(#95): Don't show this extra border in dark mode, see:
                 //   https://github.com/zulip/zulip-flutter/pull/317#issuecomment-1784311663
                 border: Border(left: BorderSide(
                   width: 1,
-                  color: Colors.white.withOpacity(0.6))))))),
+                  color: Color(0x99ffffff))))))),
```
Comment (gnprice) [inline lib/widgets/message_list.dart:506]: The `withOpacity` computation is pretty trivial, unlike `HSLColor.toColor`. So this can just remain as it is.

**Review by gnprice — COMMENTED** [review]
Thanks! I think we don't actually need an optimization here, though, because of these two the computation that's nontrivial isn't actually in a hot path.

**Comment by chrisbobbe** [comment]
Oh OK, makes sense! Thanks for the review; closing.

**Pull request closed without merging**